### ISSUE TITLE
New Resource `azuredevops_variable_group_variable`

### ISSUE THREAD

**Issue by juicybaba** [microsoft/terraform-provider-azuredevops#1003]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

I am working on a use case where terraform generates a variable group with few pre-defined variables and allow users to add additional variables as needed to the same variable group.  Sadly, manually created variables will be removed once terraform runs. 

`azuredevops_variable_group_variable` can be helpful. Even if variable has to be created with at least one variable, create a dummy variable with ignore_change on variable block and manage pre-defined `azuredevops_variable_group_variable` on the side can do the trick.

### New or Affected Resource(s)

* azuredevops_variable_group_variable

### Potential Terraform Configuration

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key.
```

### References

* #0000

**Comment by seblatre** [comment]
Hello,
I have the pretty same use case for our data platform, we are providing Variable group to our internal customers for generic stuff and they can add their own variables for their specific use cases so here we are blocked by the current provider implementation.

**Comment by drdamour** [comment]
this is a dupe of (though probably this one is stated more clearly than) #231 .  probably this one should be closed/merged into #231
